rest_framework-stubs/generics.pyi

```diff
@@ -9,6 +9,7 @@ from rest_framework.pagination import BasePagination
 from rest_framework.request import Request
 from rest_framework.response import Response
 from rest_framework.serializers import BaseSerializer
+from django_filters.rest_framework import DjangoFilterBackend
```

Comment:
Sorry, we cannot do that. Because we can't rely on third party projects like `django-filters` here.